### PR TITLE
Add new TFX OSS InfraValidator component to tfx-caip-tf21 lab-01.

### PR DESCRIPTION
**Pull request by dougkelly**
- Add InfraValidator component and explanatory text to pipeline.
- Add InfraValidator.png.
- Update Pusher.png.
- Remove references to deprecated ModelValidator.

### CODE REVIEW

File: workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb [hunk 1]
```diff
@@ -71,7 +71,7 @@
     "from tfx.components import BigQueryExampleGen\n",
```
Comment (BenoitDherin) [inline workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb:71]: <ul><li>Move this licensing stuff at the very end of the notebook</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (BenoitDherin) [inline workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb:71]: <ul><li>Put this section (along with its title) at the very top of the notebook</li><li>Add learning objectives (see <a href="https://github.com/GoogleCloudPlatform/mlops-on-gcp/blob/master/workshops/kfp-caip-sklearn/lab-01-caip-containers/lab-01.ipynb" target="_blank">this lab</a> to keep formatting consistency)</li><li>"data exploriation" -&gt; "data exploration"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (BenoitDherin) [inline workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb:71]: <p>When I run this cell in the JupyterLab, I am getting this error</p><p> </p><p><code>`</code>python</p><pre class="ql-syntax" spellcheck="false"> Error importing tfx_bsl_extension.arrow.array_util. Some tfx_bsl functionalities are not availableError importing tfx_bsl_extension.arrow.table_util. Some tfx_bsl functionalities are not availableError importing tfx_bsl_extension.arrow.table_util. Some tfx_bsl functionalities are not available
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
&lt;ipython-input-1-c95642f63133&gt; in &lt;module&gt;
     17 from tfx.components import Evaluator
     18 from tfx.components import ExampleValidator
---&gt; 19 from tfx.components import InfraValidator
     20 from tfx.components import Pusher
     21 from tfx.components import ResolverNode

ImportError: cannot import name 'InfraValidator' from 'tfx.components' (/opt/conda/lib/python3.7/site-packages/tfx/components/__init__.py)
</pre><p> </p><p><code></code>`</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (BenoitDherin) [inline workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb:71]: <ul><li>"is able to able to generate" --&gt; "is able to generate"</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='82'/>

**Comment by BenoitDherin** [comment]
<p>Please update the notebook so that it checks that all the components are installed, and if not, that it installs them. </p><p>I'll retest the notebook after that. </p>

File: workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb (hunk 1 above)
Comment (dougkelly) [reply workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb:71]: I am unable to replicate the error you are seeing. Are you using perhaps the Python runtime? I am using the Tensorflow 2.1 Enterprise version; should we prefer one over the other?

Here are the package versions I have in my environment:

Tensorflow Version: 2.1.0
TFX Version: 0.21.4
TFDV Version: 0.21.5
TFMA Version: 0.21.6

Comment (dougkelly) [reply workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb:71]: This PR is to add it InfraValidator to the lab notebook first. I branch from the lab to make the exercises in the format you suggested after this is submitted.

Comment (dougkelly) [reply workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb:71]: I will make this formatting change in a subsequent PR for the exercises notebook.

Comment (dougkelly) [reply workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb:71]: Thanks. Updated.

Comment (dougkelly) [reply workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb:71]: Added another commit for explicitly listing lab package versions and gave instructions and code to update the packages in the current Jupyter kernel.

Comment (dougkelly) [reply workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb:71]: Added commit to address data exploration typo. Thanks for catching.

Comment (BenoitDherin) [reply workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb:71]: <p>okay. Make sure that the formatting changes are the same in the exercises and the notebook.</p>

Comment (BenoitDherin) [reply workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb:71]: <p>LGTM</p>

**Comment by BenoitDherin** [comment]
<p>LGTM</p>

**Pull request merged**